homeworks/karolinavalko/4_exceptions_logging/to_do.py

```diff
@@ -0,0 +1,79 @@
+import logging
+import logging.config
+import os
+
+
+print(os.getcwd())
+
+logging.config.fileConfig(
+    r"c:\Users\tabal\OneDrive\ドキュメント\python_hu\homeworks\karolinavalko\4_exceptions_logging\logger_set_up.ini"
+)
+logger = logging.getLogger("my_logger")
+
+file_path = r"c:\Users\tabal\OneDrive\ドキュメント\python_hu\homeworks\karolinavalko\4_exceptions_logging\to_do.txt"
+to_do_list = []
+display_menu = ["Add task", "View task", "Remove task", "Exit"]
+
+
+def read_file(file_path):
```

Comment:
ebben a function-ben is szuper lenne visszaadni egy üres listát ha valami hiba történik.